docs/getting-started/self-healing-rollback.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[self-healing-rollback]]
+= Configure self-healing rollback for Windows endpoints
+
+{endpoint-cloud-sec}'s self-healing feature rolls back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features. All activity on the host reverts to its state five minutes before the prevention alert.
+
+This can help contain the impact of malicious activity, as {endpoint-cloud-sec} not only stops the activity but also erases any attack artifacts deployed prior to detection.
```

Comment:
Same question about terminology. Isn't Elastic Endpoint doing the stopping of malicious activity? 🤔

Reply:
"Elastic Endpoint" seems to me more like an internal piece of technology than something a customer directly experiences. I was using the integration name here to more generally say that _the product_ or _the integration_ stops and rolls back the attack. But @ferullo @joe-desimone etc please let us know what you think.

Reply:
the integration name seems correct here.